cmd/antrea-controller/controller.go

```diff
@@ -144,6 +146,8 @@ func run(o *Options) error {
 	egressInformer := crdInformerFactory.Crd().V1alpha2().Egresses()
 	externalIPPoolInformer := crdInformerFactory.Crd().V1alpha2().ExternalIPPools()
 	externalNodeInformer := crdInformerFactory.Crd().V1alpha1().ExternalNodes()
+	adminNPInformer := policyInformerFactory.Policy().V1alpha1().AdminNetworkPolicies()
+	banpInformer := policyInformerFactory.Policy().V1alpha1().BaselineAdminNetworkPolicies()
```

Comment:
this is a  general comment for the entire PR: I don't think an acronym like banp is very readable. I think we should stick with baselineAdminNP even if it's longer. It's also more consistent with adminNP.

Reply:
So the reason I used `adminNP` in a lot of these places, is that by the time I wrote these, https://github.com/antrea-io/antrea/pull/5081 was still not merged and the AdminNetworkPolicy shortname (which is anp) collides with that of AntreaNetworkPolicy. Now that AntreaNetworkPolicy has been re-shortnamed to `ANNP`, I supposed we can use `anp` and `banp` for these two resources now as those are the shortnames of these resources actually.